### PR TITLE
Clarify network vs neighborhood basis definitions for symmetric vs semimetric

### PR DESCRIPTION
**Pull request by StevenClontz**
Open to feedback here, but I wanted to restate these definitions to make clear that the distinction between symmetric/semimetric is that symmetric spaces are generated by a network, while semimetric spaces are generated by a neighborhood basis. Note every neighborhood basis is a network, but not vice versa.

### CODE REVIEW

File: properties/P000102.md [hunk 1]
```diff
@@ -12,14 +12,18 @@ refs:
   name: On semi-metric spaces (Wilson)
 ---
 
+A space whose topology is generated by a *neighborhood basis* of
+symmetric balls $B(x,\varepsilon)=\{y\in X:d(x,y)<\varepsilon\}$ for some symmetric $d$.
+That is, a subset $U \subseteq X$ is open iff for each $x \in U$,
+there is some $V\subseteq U$ containing $x$ such that for each $y\in V$,
+there is $\varepsilon > 0$ with $B(y,\varepsilon)\subseteq V$.
```
Comment (StevenClontz) [inline properties/P000102.md:19]: Actually, is this quite enough to guarantee that $B(x,\varepsilon)$ is a neighborhood of each $x$?

Comment (david20000813) [reply properties/P000102.md:19]: No, this is equivalent to just saying every point having a neighborhood ball around it, which is symmetrizable, not semimetrizable.

File: properties/P000104.md [hunk 2]
```diff
@@ -10,12 +10,14 @@ refs:
...
+That is, a subset $U \subseteq X$ is open iff for each $x \in U$,
+there is some $\varepsilon > 0$ with $B(x,\varepsilon)\subseteq U$.
+
 For a set $X$, a function $d:X^2\to[0,\infty)$ is called a *symmetric*
 if it satisfies reflexivity ($d(x,y)=0\Leftrightarrow x=y$) and symmetry
-($d(x,y)=d(y,x)$). A space $X$ is called *symmetrizable* if
-there is a symmetric such that a subset $U \subseteq X$ is open iff for each $x \in U$,
-there is some $\varepsilon > 0$ with $B(x,\varepsilon)=\{y\in X:d(x,y)<\varepsilon\}$
-being contained in $U$.
+($d(x,y)=d(y,x)$); in this case, we will call this symmetric a *semimetric*.
```
Comment (david20000813) [inline properties/P000104.md:20]: I think you have mistakenly put this line in symmetrizable instead of semimetrizable?

**Comment by prabau** [comment]
The previous formulation seemed clear and unambiguous to me. I you want to mention these points, I think it would be better to add separate paragraphs about it instead of changing the previous text.

**Comment by StevenClontz** [comment]
Thanks for the feedback. Marking as draft and will take another pass.

**Comment by StevenClontz** [comment]
> The previous formulation seemed clear and unambiguous to me. I you want to mention these points, I think it would be better to add separate paragraphs about it instead of changing the previous definition per se.

My write up definitely needs improvement, but I do think that the cleanest distinction here is whether the symmetric is used to create a network-generated vs basis-generated topology. That said, I'm okay with delegating the characterization to an "equivalently" list.

**Pull request closed without merging**